package.json

```diff
@@ -2,18 +2,19 @@
   "name": "@govtechsg/open-attestation",
   "version": "0.0.0-development",
   "description": "## Usage",
-  "main": "dist/cjs/index.js",
```

Comment:
do we want to provide main for [backward compatibility](https://2ality.com/2021/06/typescript-esm-nodejs.html#an-entry-point-for-the-package-itself)?
```
If both "exports" and "main" are defined, the "exports" field takes precedence over "main". "exports" are not specific to ES modules or CommonJS; "main" is overridden by "exports" if it exists. As such "main" cannot be used as a fallback for CommonJS but it can be used as a fallback for legacy versions of Node.js that do not support the "exports" field.
```
https://nodejs.org/api/packages.html#packages_package_entry_points

Reply:
Nvm just saw your `.nvmrc` file

Reply:
Also i realise your engine version is different from the node version specified in .nvmrc

Reply:
mostly dropped node 10 support so no need